front/app/containers/App/constants.ts

```diff
@@ -49,38 +49,6 @@ export const locales = [
   'sr-SP',
 ];
 
-export const graphqlLocales = [
```

Comment:
Normally, you should be able to delete this from constants.js as well then

Reply:
Makes sense!